traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -27,7 +27,7 @@
 
 
 public class Router {
-    private static final String BASE_URL = "https://traveler.rc.guestlogix.io/v1";
+    private static final String BASE_URL = BuildConfig.API_ENDPOINT;
```

Comment:
There needs to be default value

The goal here was that the SDK reads from SharedPrefs. not BuildConfig...BuildConfig is an internal config file and clients should not have to set it

Reply:
Lets chat about this